mmdet3d/core/visualizer/show_result.py

```diff
@@ -131,7 +131,7 @@ def show_seg_result(points,
                     filename,
                     palette,
                     ignore_index=None,
-                    show=False,
```

Comment:
Why this PR modify this?

Reply:
Oh, that's because when I benchmark the model, I want to do some visualization to help me decide if the output is correct. Then I realized the previous code has this bug so I fix it in this PR. The order here will cause error when setting `--show` in `test.py` args.

Reply:
I just want to make it consistent with `show_result()`. In my opinion, `show_result()`, `show_seg_result()` and `show_multi_modality_result()` are three similar visualization functions in the same file, we should keep their default args consistent. As `show_result()` has `True` for default value of `show`, I am changing the other two to `True`, too.

I have checked the codebase and this won't cause any BC-breaking. `show` args is always assigned when calling this function.

Reply:
I agree that this shouldn't be modified in this PR. I am changing it back. Maybe do these modifications in a new PR.

Reply:
Changed them in #783